src/diffusers/pipelines/flux/pipeline_flux.py

```diff
@@ -501,8 +505,10 @@ def prepare_latents(
         generator,
         latents=None,
     ):
-        height = int(height) // self.vae_scale_factor
-        width = int(width) // self.vae_scale_factor
+        # VAE applies 8x compression on images but we must also account for packing which requires
+        # latent height and width to be divisible by 2.
+        height = int(height) // self.vae_scale_factor - ((int(height) // self.vae_scale_factor) % 2)
```

Comment:
is this same as this? (basically revert back to the original code but divisible by `self.vae_scale_factor*2`
a little bit easier to understand, no?
```python
height = 2 * (int(height) // (self.vae_scale_factor*2))
```